app/screens/redeem-lnurl-withdrawal-screen/redeem-bitcoin-detail-screen.tsx

```diff
@@ -135,17 +140,28 @@ const RedeemBitcoinDetailScreen = ({
   navigation,
   route,
 }: StackScreenProps<RootStackParamList, "redeemBitcoinDetail">) => {
-  const { fiatSymbol } = useDisplayCurrency()
+  const { formatMoneyAmount, displayCurrency } = useDisplayCurrency()
 
   const { callback, domain, defaultDescription, k1, minWithdrawable, maxWithdrawable } =
     route.params.receiveDestination.validDestination
-  const minWithdrawableSatoshis = minWithdrawable / 1000
-  const maxWithdrawableSatoshis = maxWithdrawable / 1000
+  const minWithdrawableSatoshis: BtcPaymentAmount = {
+    amount: Math.round(minWithdrawable / 1000),
```

Comment:
is `minWithdrawable` in msats? maybe this should be explained as why we're divided by 1000

Reply:
I can add a comment